week-1-practise/level-1/8-function-output-1.js

```diff
@@ -6,7 +6,7 @@ var result = getNumber();
 console.log(result);
 
 // Answer these questions:
-// - What does this program do?
-// - How many parameters does the function take?
-// - What is the function name?
-// - Where is the function called? with what parameters?
\ No newline at end of file
+// - What does this program do? It takes a random number and multiplies it by 10 and later it's printed the result
+// - How many parameters does the function take? 1
+// - What is the function name? getNumber
+// - Where is the function called? with what parameters? It's called in line 5 after the = symbol and it's called with an undefined parameter.
```

Comment:
You could say that, but the more common way to describe this would be that the function is called with no parameters.